.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Test sros2
+on:
+  pull_request:
+  push:
+  schedule:
+    # Run daily
+    - cron:  '0 12 * * *'
+
+jobs:
+  build_and_test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
```

Comment:
Given the ROS 2 release strategy, should this actually be `ubuntu-18.04`? This question applies to the linters, as well.

Reply:
As this is meant to be the day-to-day development I was thinking that latest was better so that we can switch to 20.04 automatically.
Granted IDK when Github Actions will provide a more recent runner while ROS2 CI should migrate shortly. Whichraises the question of how valid the ros-ci-action approach will be if Github doesnt provide a 20.04 environment in the near future.

I'll switch to ubuntu-18-04 for now and move forward with the actions approach and we can revisit in a couple months.